scripts/us_census/acs5yr/subject_tables/README.md

```diff
@@ -0,0 +1,64 @@
+## Importing American Community Survey (ACS) Subject Tables
+
+This directory has the files and code used for importing ACS Subject Tables into DataCommons.
+
+Note: Before running the scripts here ensure the packages specified in `data/requirements.txt` is installed.
+
+### Getting Started
+Before, getting started with an import of an ACS Subject Table, it is important to ensure the following are available:
+1. [**JSON Specification**](jsonSpec.md) describing the values that are applicable for a property based on a token (substring or part) of the column name.
+
+The JSON Specification also defines the default defintions, and other properties for a statistical variable. The [jsonSpec.md](jsonSpec.md) has a detailed description of how to defined a new JSON specification.
+
+2. Running the script:
+```shell
+  python3 process.py <args>
+```
+
+We have a unified script that can be used to generate either the column map
+or process data or doing both the steps. The mode at which the script is run is
+set by command-line args. The following are the command line flags supported:
+```
+Generic proces module to generate the csv/tmcf and csv
+flags:
+
+process.py:
+  --[no]debug: [for processing]set the flag to add additional columns to debug
+    (default: 'false')
+  --[no]has_percent: [for processing]Specify the datasets has percentage values that needs to be convered to counts
```

Comment:
Can presence of a non-empty 'measurementDenominator' in the spec be an indicator for converting percentages to counts?